core/consensus/rhd/src/lib.rs

```diff
@@ -494,7 +494,7 @@ pub struct BftService<B: Block, P, I> {
 	live_agreement: Mutex<Option<(B::Header, AgreementHandle)>>,
 	round_cache: Arc<Mutex<RoundCache<B::Hash>>>,
 	round_timeout_multiplier: u64,
-	key: Arc<ed25519::Pair>, // TODO: key changing over time.
+	key: Arc<ed25519::Pair>, // FIXME: key changing over time.
```

Comment:
```suggestion
	key: Arc<ed25519::Pair>,
```
removed. RHD is currently deactivated and needs to be refactored anyways.